moveit_core/transforms/src/transforms.cpp

```diff
@@ -37,17 +37,22 @@
 #include <moveit/transforms/transforms.h>
 #include <tf2_eigen/tf2_eigen.h>
 #include <boost/algorithm/string/trim.hpp>
-#include <ros/console.h>
+#include "rclcpp/rclcpp.hpp"
 
 namespace moveit
 {
 namespace core
 {
+
+// Logger
+rclcpp::Logger logger_transforms = rclcpp::get_logger("transforms");
```

Comment:
I know this has been debated elsewhere so forgive me if it has already been agreed upon:

It seems less than ideal to have the logger initialized here. (see: http://wiki.ros.org/CppStyleGuide#Globals, 
https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables)

It would be better to use the same approach as the [robot_state.cpp](https://github.com/ros-planning/moveit/blob/211158d00a6938c819f5d51301662b9d993b0c19/moveit_core/robot_state/src/robot_state.cpp#L58) where we declare the LOGNAME string at the top of the source file (`const std::string LOGNAME = "transforms";`) that is used throughout.

In this case, I don't see a reason not to use a class variable `rclcpp::Logger logger_transforms_` that gets set on initialization and used throughout the class.

In general, is not possible to use a logger class variable, it would be better for each `RCLCPP_ERROR` to create it's own logger. 

```cpp
RCLCPP_ERROR(rclcpp::get_logger(LOGNAME), "...");
```

Reply:
please keep this discussion in the unified issue:
https://github.com/ros-planning/moveit2/issues/21

Reply:
You beat me to it. I was just about to move this comment to that issue. :)